lcls_tools/common/devices/yaml/GUNB.yaml

```diff
@@ -22,6 +22,7 @@ magnets:
       - SC_STMO
       - SC_STXI
       - SC_SXR
+      l_eff: 0.0
```

Comment:
This should be the same length as CQ02B? Not sure why this one is returning 0.0. We should look into this.

Reply:
I spot checked on CQ01B it is  and the entry is "" in the GUNB.yaml
Which in my code is set to return 0.0.
I will check a few more of these, however I am observing a lot of "" fields in he lcls_elements.csv